nuxeo-coldstorage-web/nuxeo-coldstorage.html

```diff
@@ -35,3 +35,9 @@
     ></nuxeo-move-contents-to-coldstorage-button>
   </template>
 </nuxeo-slot-content>
+
+<nuxeo-slot-content name="coldStorageDocumentInfoBar" slot="DOCUMENT_INFOBAR">
+  <template>
+    <nuxeo-coldstorage-infobar document="[[document]]"></nuxeo-coldstorage-infobar>
+  </template> </nuxeo-slot-content
```

Comment:
The formatting seems to be a bit off here.

Reply:
indeed. fixed